### PR TITLE
Feature/airport entity

### PR DESCRIPTION
**Pull request by xkrejci7**
Please review my Airport entity:
* Airport Entity
* Airport DAO Interface

### CODE REVIEW

File: src/main/java/cz/muni/airport/model/MainAirport.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+package cz.muni.airport.model;
+
+import java.util.Locale;
+
+/**
+ *
+ * @author Jirka
+ */
+public class MainAirport {
```
Comment (Richardb953) [inline src/main/java/cz/muni/airport/model/MainAirport.java]: metodu main mame iba jednu a ta bude spolocna... toto prosim ta zmaz.

File: src/main/java/cz/muni/airport/dao/IAirportDao.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import java.util.List;
+import java.util.Locale;
+
+/**
+ *
+ * @author Jiri Krejci
+ */
+public interface IAirportDao {
+    
+    	public void create(Airport airport) throws IllegalArgumentException;
```
Comment (Richardb953) [inline src/main/java/cz/muni/airport/dao/IAirportDao.java]: v interface nemusis davat public, chyba to nie je ale nie je to povinne, interface je vzdy cely public :) len do buducna aby si vedel

File: src/main/java/cz/muni/airport/model/MainAirport.java (hunk 1 above)
Comment (xkrejci7) [inline src/main/java/cz/muni/airport/model/MainAirport.java]: OK,mazu

File: src/main/java/cz/muni/airport/dao/IAirportDao.java (hunk 2 above)
Comment (xkrejci7) [inline src/main/java/cz/muni/airport/dao/IAirportDao.java]: OK, mazu public, dik

**Pull request merged**